bin/rucio-dark-reaper

```diff
@@ -33,11 +34,13 @@ def get_parser():
     """
     parser = argparse.ArgumentParser(description="The Dark-Reaper daemon is responsible for the deletion of quarantined replicas.")
     parser.add_argument("--run-once", action="store_true", default=False, help='One iteration only')
-    parser.add_argument("--all-rses", action="store_true", default=False, help='Select RSEs from the quarantined queues')
```

Comment:
The removal of this option is a bit problematic. I understand why you do it, but adding this in a patch release is a bit problematic, since this might fail unexpectedly for some communities.
Can you maybe add this in two steps.
1. In this PR you still leave the `--all-rses` in, but if used in multi-vo mode you give an error. You also add the new arguments (exclude-rses, include-rses, vos)
2. Once this one is merged, you preapre a feature PR which removes the `--all-rses`. 

Not only for the dark-reaper but also for the other daemons where changed.